<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>THelper</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox_fromlanguage">
        <item>
         <property name="text">
          <string>自动检测</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>英语</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>中文</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>粤语</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_ico_convert">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/convert.svg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_tolanguage">
        <item>
         <property name="text">
          <string>中文</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>粤语</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>英语</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_translate">
        <property name="text">
         <string>翻译</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEdit_intputtext">
        <property name="toolTip">
         <string>输入文字</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_outputtext"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
